calyx-py/calyx/builder.py

```diff
@@ -99,8 +99,14 @@ def comb_group(self, name: str) -> "GroupBuilder":
         return builder
 
     def cell(
-        self, name: str, comp: ast.CompInst, is_external=False, is_ref=False
+        self, name: str, comp: Union[ast.CompInst, "ComponentBuilder"],
+        is_external=False, is_ref=False
     ) -> "CellBuilder":
```

Comment:
How can this feature be used? Or maybe, how does it show up in code?

Reply:
It's for instantiating components that you just recently built with the builder! Like this in AOC day 1:
https://github.com/cucapra/aoc2022-calyx/blob/e63cff9055fa7e183dabab6993f2dee26376e6ab/1/accelgen.py#L103

Reply:
Awesome! Makes sense